en_ID/pages/interns.md

```diff
@@ -26,7 +26,6 @@ Put images link below
 [fatwadwihidayat-img]: https://github.com/FatwaDwiHidayat.png?size=25
 [hananaskarim-img]: https://github.com/HananAskarim.png?size=25
 [MuhammadAzka9-img]:https://github.com/MuhammadAzka9.png?size=25
-
```

Comment:
removal of this new line is also not necessary, when creating a PR we should focus on the changes required.

Reply:
i have returned that line